HW3/index.js

```diff
@@ -0,0 +1,39 @@
+const User = require("./models/user");
+const Worker = require("./models/worker");
+const Platform = require("./models/platform");
+const UserService = require("./services/user-service");
+
+console.log("Hello World!");
+
+async function main() {
+  const workForce = new Platform("WORKFORCE");
+  const haris = new User("Haris", 35, "haris@xyz.com");
+  const sam = new User("Sam", 34, "sam@xyz.com");
+  const tyler = new Worker("Tyler", "Plumbing", 5, "tyler@xyz.com");
+  const langard = new Worker("Langard", "Electrician", 3, "langard@xyz.com");
+
+  workForce.registerUser(sam);
+  workForce.registerUser(haris);
+
+  workForce.registerWorker(tyler);
+  workForce.registerWorker(langard);
+
+  workForce.report();
+
+  await UserService.add(sam);
+  await UserService.add(haris);
+
+  const user = await UserService.findAll();
+
+  console.log(user[0].name);
+  console.log(user[1].name);
+
+  await UserService.del(1);
+
+  const newuser = await UserService.findAll();
+
+  console.log("User with id 1 has been deleted  the remaining users are: ");
+  console.log(newuser[0].name);
```

Comment:
One line above you indicated "the remaining users are:" but you are printing only the name of first user in this array.